src/test/ui-fulldeps/hash-stable-is-unstable.rs

```diff
@@ -12,4 +12,5 @@ use rustc_macros::HashStable;
 
 #[derive(HashStable)]
 //~^ use of unstable library feature 'rustc_private'
+//~| use of unstable library feature 'rustc_private'
```

Comment:
is there a reason for this duplicate line?

Reply:
Those directives match the start of an error message. Both error messages start the same, but then contain a different crate being loaded from the sysroot: `crate "rustc_data_structures" is being from the sysroot, [...]` and `crate "rustc" is being from the sysroot, [...]`.